xbmc/video/VideoInfoScanner.cpp

```diff
@@ -472,10 +472,16 @@ namespace VIDEO
           mediaType = MediaTypeTvShow;
         else if (info2->Content() == CONTENT_MUSICVIDEOS)
           mediaType = MediaTypeMusicVideo;
-        CServiceBroker::GetEventLog().Add(EventPtr(new CMediaLibraryEvent(
-          mediaType, pItem->GetPath(), 24145,
-          StringUtils::Format(g_localizeStrings.Get(24147).c_str(), mediaType.c_str(), URIUtils::GetFileName(pItem->GetPath()).c_str()),
-          pItem->GetArt("thumb"), CURL::GetRedacted(pItem->GetPath()), EventLevel::Warning)));
+
+        if (info2->Content() == CONTENT_TVSHOWS)
```

Comment:
Maybe the whole logic could be simplified by first retrieving the proper path either for tvshows or for movies / musicvideos and then use the same call to `CServiceBroker::GetEventLog().Add()` using the previously retrieved path.

Reply:
Yes, I agree. If we can do that with `CMediaTypes::GetLocalization(mediaType)` instead of checking via `info2->Content()` then, yes. 

As said, I'm not a developer. I can try and see where I will end up. But I have no worries if someone else with more c++ foo will open another PR. I will happily close this one if we have something better ;)

Reply:
Sorry I meant the part below. This part can probably not be improved.

Reply:
I guess, I'm stuck at "simplifying the logic". Not saying that it's not possible. But that might be out of my capabilities. If you feel it's important, then I might need a helping hand. Feel free to PR something against my branch and I'll happily merge it